learningresources/api.py

```diff
@@ -301,3 +305,44 @@ def import_static_assets(course, path):
                 name = join(root, name).replace(path + sep, '', 1)
                 django_file.name = name
                 create_static_asset(course.id, django_file)
+
+
+def join_description_paths(*args):
+    """
+    Helper function to format the description path.
+    Args:
+        list of dpath (unicode): description path
+    Returns:
+        formatted dpath (unicode)
+    """
+    formatted_dpath = join(
+        *[' {0} '.format(dpath) for dpath in args if dpath != '']
+    )
+    return formatted_dpath.strip()
+
+
+def update_description_path(resource, force_parent_update=False):
```

Comment:
Would it make sense to move this function to a method in `LearningResource`? The only required argument is the `resource` model, it doesn't return anything and mutates the `LearningResource`